kairon/events/definitions/mail_channel_schedule.py

```
@@ -0,0 +1,51 @@
+
+    def enqueue(self, **kwargs):
+        """
+        Send event to event server.
+        """
+        try:
+            mails: list = kwargs.get('mails', [])
+            payload = {'bot': self.bot, 'user': self.user, 'mails': mails}
+            Utility.request_event_server(EventClass.email_channel_scheduler, payload)
+        except Exception as e:
+            logger.error(str(e))
+            raise AppException(e)
```

Ruff (0.8.0)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)